CoreNotesView/src/au/gov/asd/tac/constellation/views/notes/NotesViewTopComponent.java

```diff
@@ -116,7 +116,7 @@ protected void handleComponentOpened() {
     @Override
     protected void handleComponentClosed() {
         super.handleComponentClosed();
-        notesViewPane.closeEdit();
+        //notesViewPane.selectAllFilters();
```

Comment:
Given the PR is going into `master` please remove dead commented code if you don't need it.